src/codegen/reverse.jl

```diff
@@ -1,16 +1,20 @@
 # Codegen shared by both stage1 and stage2
 
-function make_opaque_closure(interp, typ, name, meth_nargs::Int, isva, lno, cis, revs...)
+function make_opaque_closure(interp, typ, name, meth_nargs::Int, isva, lno, ci, revs...)
     if interp !== nothing
-        cis.inferred = true
+        @static if VERSION ≥ v"1.12.0-DEV.15"
+            rettype = Any # ci.rettype # TODO revisit
```

Comment:
Why? is inference broken?

Reply:
`ci` no longer has `rettype`.

Reply:
We probably need further tweak on this code, but maybe later.